src/main/java/com/ultimatesoftware/dataplatform/vaultjca/VaultAuthenticationLoginCallbackHandler.java

```diff
@@ -120,12 +121,22 @@ public void handle(Callback[] callbacks) throws IOException, UnsupportedCallback
     }
   }
 
+  private boolean isTemplatedPath(String path) {
+    return path.contains("{username}");
```

Comment:
nit: should the `{username}` string be declared as a constant ? It's used multiple times in the code

Reply:
Will do.